controllers/pullRequestsController.js

```diff
@@ -14,14 +14,17 @@ const getPRdetails = async (req, res) => {
 
     if (data.total_count) {
       const allPRs = []
-      data.items.forEach(({ title, html_url: url, state, created_at: createdAt, updated_at: updatedAt, draft, labels, assignees }) => {
+      data.items.forEach(({ title, html_url: url, state, created_at: createdAt, updated_at: updatedAt, repository_url: repositoryUrl, draft, labels, assignees }) => {
         const allAssignees = assignees.map(object => object.login)
         const allLabels = labels.map(object => object.name)
+        const repositoryUrlSplit = repositoryUrl.split('/')
+        const repositoryName = repositoryUrlSplit[repositoryUrlSplit.length - 1]
         allPRs.push({
           title,
           state,
           createdAt,
           updatedAt,
+          repositoryName,
```

Comment:
`repositoryName` seems too lengthy... What about `repository`? Short and conveys the meaning

Reply:
Sure.